airbyte-cdk/python/airbyte_cdk/sources/streams/http/requests_native_auth/abstract_oauth.py

```diff
@@ -77,7 +77,7 @@ def refresh_access_token(self) -> Tuple[str, int]:
         """
         try:
             response_json = self._get_refresh_access_token_response()
-            return response_json[self.get_access_token_name()], response_json[self.get_expires_in_name()]
```

Comment:
This change (and others in the CDK) are Included in #20301

Reply:
@joelluijmes can you make the base branch for this the one in #20301 to accurately understand the diff this is introducing? (it will be automatically changed to master once #20301 is merged)